### PR TITLE
Menu ads

### PR DESCRIPTION
**Pull request by marcelinkaaa**
Added another instance of 'remove ad' call to action on the bottom of the navigation drawer

### CODE REVIEW

File: android/res/drawable/ad_menu_item_background.xml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+-->
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="rectangle">
+    <solid android:color="@color/transparent"/>
+    <stroke android:color="@color/app_highlight_text" android:width="2dp"/>
+    <corners android:radius="3dp"/>
+</shape>
```
Comment (aldenml) [inline android/res/drawable/ad_menu_item_background.xml]: \n

File: android/src/com/frostwire/android/gui/views/AdMenuItemView.java [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+        View.inflate(getContext(), R.layout.view_ad_menuitem, this);
+
+        textHeadline = (TextView) findViewById(R.id.view_ad_menu_item_headline);
+        textSubtitle = (TextView) findViewById(R.id.view_ad_menu_item_subtitle);
+        textThumbnail = (TextView) findViewById(R.id.view_ad_menu_item_thumbnail);
+        imageThumbnail = (ImageView) findViewById(R.id.view_ad_menu_item_thumbnail_image);
+
+        Random myRand = new Random();
+        myRand.nextInt();
```
Comment (aldenml) [inline android/src/com/frostwire/android/gui/views/AdMenuItemView.java]: remove this line

File: android/src/com/frostwire/android/gui/views/AdMenuItemView.java [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+        boolean isEven = (myRand.nextInt() % 2) == 0;
+
+        if (isEven == true) {
```
Comment (gubatron) [inline android/src/com/frostwire/android/gui/views/AdMenuItemView.java]: next time just do:

`if (isEven) {`

no need to compare to `true` the `if` always checks against `true`

File: android/src/com/frostwire/android/gui/views/AdMenuItemView.java [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+            textHeadline.setText(R.string.support_frostwire);
+            textSubtitle.setText(R.string.save_bandwidth);
+            textThumbnail.setVisibility(VISIBLE);
+            textThumbnail.setText(R.string.ad_free);
+        } else {
+            textHeadline.setText(R.string.support_frostwire);
```
Comment (gubatron) [inline android/src/com/frostwire/android/gui/views/AdMenuItemView.java]: `textHeadline.setText(R.string.support_frostwire);`

can be outside the `if/else`, no need to repeat the code if it happens either way.

**Comment by gubatron** [comment]
I've rebased all of this and created a new `menu-ads-rebased` branch on `marcelinkaaa`'s repo.
I'll close this PR for now, and will continue to work on the rebased branch.

**Pull request closed without merging**